sui/its.js

```diff
@@ -56,6 +63,9 @@ if (require.main === module) {
         .name('setup-trusted-address')
         .description('Setup trusted address')
         .command('setup-trusted-address <trusted-chain> <trusted-address>')
+        .addOption(new Option('--offline', 'store tx block for offline signing'))
+        .addOption(new Option('--sender <sender>', 'transaction sender'))
+        .addOption(new Option('--txFilePath <file>', 'unsigned transaction will be stored'))
```

Comment:
Perhaps create a utility function to add these options, similar to `addBaseOptions` ?